pkg/webhook/server/server.go

```diff
@@ -55,8 +55,8 @@ var (
 
 func init() {
 	apiextensionsinstall.Install(defaultScheme)
-	admissionv1beta1.AddToScheme(defaultScheme)
-	admissionv1.AddToScheme(defaultScheme)
+	_ = admissionv1beta1.AddToScheme(defaultScheme)
+	_ = admissionv1.AddToScheme(defaultScheme)
```

Comment:
thought: I think discarding errors is actually wrong here, although nothing will work if adding to scheme fails anyway. In other controllers I've seen:

`import "k8s.io/apimachinery/pkg/util/runtime"`
```suggestion
	runtime.Must(admissionv1beta1.AddToScheme(defaultScheme))
	runtime.Must(admissionv1.AddToScheme(defaultScheme))
```

Reply:
```bash
$ go doc -src "k8s.io/apimachinery/pkg/util/runtime" Must
package runtime // import "k8s.io/apimachinery/pkg/util/runtime"

// Must panics on non-nil errors. Useful to handling programmer level errors.
func Must(err error) {
        if err != nil {
                panic(err)
        }
}
```

I was expecting more than that from the impl, somehow :joy: 

I did consider panicking on this, because as you say if this function fails we're probably doomed... the reason I didn't was that panicking could be a breaking change, even if it's a good change to make.

If you're comfortable with it (and obviously you are given you suggested the code :joy: ) I'll make the change because I think it's worth it.